.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
We usually use `daily`, which seems like it'd make more sense for the problem we're trying to solve.

Reply:
Thanks, filed https://github.com/coreos/fedora-coreos-stream-generator/pull/28. At first I hadn't realized that it was the only module so didn't want Dependabot to be too spammy.